staging/src/k8s.io/client-go/rest/request.go

```diff
@@ -831,9 +846,9 @@ func glogBody(prefix string, body []byte) {
 		if bytes.IndexFunc(body, func(r rune) bool {
 			return r < 0x0a
 		}) != -1 {
-			glog.Infof("%s:\n%s", prefix, hex.Dump(body))
+			glog.Infof("%s:\n%s", prefix, truncateBody(hex.Dump(body)))
```

Comment:
Could you add a prefix like `[truncated]`?

Reply:
suffix? Yeah very good idea!